Sources/Fullscreen/Stories/Subviews/FeedbackCollectionViewCell.swift

```diff
@@ -0,0 +1,242 @@
+        view.backgroundColor = .primaryBlue
+        view.layer.cornerRadius = .spacingM
+        view.clipsToBounds = true
+        return view
+    }()
+
+    private lazy var closeButton: UIButton = {
+        let button = UIButton(withAutoLayout: true)
+        button.tintColor = .milk
+        button.setImage(UIImage(named: .close).withRenderingMode(.alwaysTemplate), for: .normal)
+        button.imageEdgeInsets = UIEdgeInsets(vertical: .spacingM, horizontal: .spacingM)
+        button.addTarget(self, action: #selector(handleCloseButtonTap), for: .touchUpInside)
+        return button
+    }()
+
+    private lazy var titleLabel: Label = {
+        let label = Label(style: .title2, withAutoLayout: true)
+        label.textColor = .milk
+        label.numberOfLines = 0
+        return label
+    }()
+
+    private lazy var optionsStackView = UIStackView(axis: .vertical, spacing: .spacingS, withAutoLayout: true)
+
+    private lazy var disclaimerLabel: Label = {
+        let label = Label(style: .detail, withAutoLayout: true)
+        label.textColor = .milk
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        return label
+    }()
+
+    private lazy var successLabel: Label = {
+        let label = Label(style: .title1, withAutoLayout: true)
+        label.textColor = .milk
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        return label
+    }()
+
+    private var optionButtons = [OptionButton]()
+
+    weak var delegate: FeedbackCollectionViewCellDelegate?
+
+    // MARK: - Init
+
+    override init(frame: CGRect) {
+        super.init(frame: .zero)
+        setup()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Setup
+
+    private func setup() {
+        contentView.addSubview(containerView)
+
+        let backgroundFigureLeft = createBackgroundImageView(with: UIImage(named: .backgroundFigureLeft))
+        let backgroundFigureTop = createBackgroundImageView(with: UIImage(named: .backgroundFigureTop))
+        let backgroundFigureRight = createBackgroundImageView(with: UIImage(named: .backgroundFigureRight))
+
+        containerView.addSubview(backgroundFigureLeft)
+        containerView.addSubview(backgroundFigureTop)
+        containerView.addSubview(backgroundFigureRight)
+
+        containerView.addSubview(closeButton)
+        containerView.addSubview(titleLabel)
+        containerView.addSubview(optionsStackView)
+        containerView.addSubview(disclaimerLabel)
+
+        NSLayoutConstraint.activate([
+            containerView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            containerView.topAnchor.constraint(equalTo: contentView.topAnchor),
+            containerView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+            // This constant aligns the background with the image background for stories.
```

Comment:
Comment about the "magic number" is good 👍 

But it would be great if there was a way to extract it to a static property somewhere that can be used both places

Reply:
I was able to extract all the constants used to calculate this constant 👍